packages/components/src/actions.ts

```diff
@@ -791,22 +799,31 @@ export function exportRows(
     form.trigger('submit');
 }
 
+export function getQueryModelExportParams(model: QueryModel, type: EXPORT_TYPES, advancedOptions?: Record<string, any>) : any {
```

Comment:
I would put this in `QueryModel/utils.ts`, which is where I have put other similar types of utils/helpers.